google-cloud-bigtable-emulator/src/main/resources/META-INF/native-image/com.google.cloud/google-cloud-bigtable-emulator/resource-config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "resources":{
+    "includes":[
+      {
+        "pattern":"\\Qgcloud/bigtable-linux-x86_64/platform/bigtable-emulator/cbtemulator\\E"
+      }
+    ]},
+  "bundles":[]
+}
```

Comment:
Not sure how graalvm works, but it looks like this will restrict this wrapper to just x64 linux. Currently it supports {darwin,linux,windows}_{x64,aarch64}.

Reply:
Memo:

```
suztomo@suztomo-macbookpro.roam.internal ~/cloud-opensource-java
~/cloud-opensource-java $ jar tf ~/Downloads/google-cloud-bigtable-emulator-0.143.2.jar |grep cbtemulator
gcloud/bigtable-darwin-arm/platform/bigtable-emulator/cbtemulator
gcloud/bigtable-linux-x86_64/platform/bigtable-emulator/cbtemulator
gcloud/bigtable-darwin-x86_64/platform/bigtable-emulator/cbtemulator
gcloud/bigtable-linux-x86/platform/bigtable-emulator/cbtemulator
gcloud/bigtable-windows-x86_64/platform/bigtable-emulator/cbtemulator.exe
gcloud/bigtable-linux-arm/platform/bigtable-emulator/cbtemulator
gcloud/bigtable-windows-x86/platform/bigtable-emulator/cbtemulator.exe
```

Reply:
Thank you both for the insights! I used a tool called the [trace-agent](https://www.graalvm.org/22.0/reference-manual/native-image/Agent/) which provides hints on the configurations needed and it looks like the suggestions it provided were restricted to just my machine's architecture. I have [added the wrappers for the architectures supported by GraalVM](https://www.graalvm.org/22.0/docs/introduction/#:~:text=GraalVM%20is%20available%20for%20Linux,on%20ARM%2064%2Dbit%20system).

Reply:
My thinking yesterday was that google-cloud-bigtable-emulator is not for users' deployed application and thus users don't expect it to work in native-image.

Today's thinking is that if a user writes a Java test with the emulator and wants to confirm the test runs on native-image, it's helpful for them if this google-cloud-bigtable-emulator module works out-of-the-box.